android-interop-testing/src/main/java/io/grpc/android/integrationtest/TesterOkHttpChannelBuilder.java

```diff
@@ -45,48 +37,26 @@ public static ManagedChannel build(
       @Nullable InputStream testCa) {
     ChannelCredentials credentials;
     if (useTls) {
-      try {
-        credentials = SslSocketFactoryChannelCredentials.create(getSslSocketFactory(testCa));
-      } catch (Exception e) {
-        throw new RuntimeException(e);
+      if (testCa == null) {
+        credentials = TlsChannelCredentials.create();
+      } else {
+        try {
+          credentials = TlsChannelCredentials.newBuilder().trustManager(testCa).build();
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
       }
     } else {
       credentials = InsecureChannelCredentials.create();
     }
 
-    ManagedChannelBuilder<?> channelBuilder = Grpc.newChannelBuilderForAddress(
```

Comment:
WIth this change there will be no interop test for `Grpc.newChannelBuilderForAddress` on Android with OkHttp?

Reply:
My earlier creds change swapped to `Grpc.newChannelBuilderForAddress`. I'm not too concerned about whether it is used/not. Before my earlier change [it was using `ManagedChannelBuilder.forAddress()`](https://github.com/grpc/grpc-java/blob/5a687e3da8fce41b51b9ffe73c8bba6f9c8a053e/android-interop-testing/src/main/java/io/grpc/android/integrationtest/TesterOkHttpChannelBuilder.java), and then casting the result to OkHttpChannelBuilder.

The only reason I'm using OkHttpChannelBuilder is to force okhttp to be used. I guess I could just do an instanceof check on the value returned by `Grpc.newChannelBuilderForAddress`... Sure. I'll do that.